rollups/evm/oracle/README.md

```
@@ -0,0 +1,66 @@
+```
+
+### Anvil
+
+```shell
+$ anvil
+```
+
+### Deploy
+
+```shell
+$ forge script script/Counter.s.sol:CounterScript --rpc-url <your_rpc_url> --private-key <your_private_key>
```

markdownlint-cli2 (0.17.2)

Dollar signs used before commands without showing output

(MD014, commands-show-output)